src/dash-playlist-loader.js

```diff
@@ -327,6 +329,8 @@ export default class DashPlaylistLoader extends EventTarget {
         this.masterLoaded_ = Date.now();
       }
 
+      this.srcUrl = resolveManifestRedirect(this.handleManifestRedirects, this.srcUrl, req);
```

Comment:
is this the only place that this needs to be done? What about `refreshXml`?

Reply:
I think it wouldn't hurt (other than adding some complexity) *but* I don't think it's necessary.  You should already be on the redirected manifest by the time you're refreshing it.  The redirect is really only going to happen on an initial load of an asset.

Reply:
I guess it's more of an issue when loading a "media" playlist which could be the sidx in a later PR. So, not a huge problem, but we'll have to deal with extra complexity at some point

Reply:
I haven't committed it yet, but I'm going to add a comment to suggest that this should be resolved due to sidx.